### PR TITLE
Lavaland Crashed escape pod ruin

### PR DESCRIPTION
**Pull request by nyozzl**
## About The Pull Request

Lavaland ruin with FISH
## Why It's Good For The Game

Lavaland fishing spot (FISH)

image

image

### CODE REVIEW

**Comment by nyozzl** [comment]
oh wait shit its stale

**Comment by Ghommie** [comment]
Ok, nice, but I'm a bit icky about placing a fishing spot right beside a tendril, which, upon being destroyed, will also leave a sizeable gap in the middle of the ruin.

**Comment by nyozzl** [comment]
i can move the tendril

**Comment by Ghommie** [comment]
whatever you prefer as long as it won't be a big issue.

**Comment by nyozzl** [comment]
replaced tendril with campfire and added can of worms deluxe and a dropped pka

**Comment by nyozzl** [comment]
heh?

**Comment by Ghommie** [comment]
> heh?

It's totally unrelated to your PR. just something we've been having since the wallening was merged hastily without the CI passing. We know what's causing it but it's going to take a while to get to the roots of the problem since any quick fix right now would be a band-aid

TL;DR don't worry, your PR is ok.

**Comment by nyozzl** [comment]
heh? tendril?.... did i fuck up? isnt there ment to be a campfire

**Comment by nyozzl** [comment]
image

**Comment by nyozzl** [comment]
PANIC PANIC PANIC

**Comment by Ghommie** [comment]
Don't jut go "PANIC PANIC", make a new PR to fix the issue. Also why is sand cluttering all of the screenshot? It looks awful.

**Comment by DaCoolBoss** [comment]
Goliath digging behaviour probably. They should have a cooldown on digging - is this after significant server run time? If they've just been given a lot of time to do this then it's normal. If they haven't, something's stopping that cooldown from happening.

**Comment by DaCoolBoss** [comment]
Double checked, what got merged isn't the updated campfire version on the screenshots. It's an older version of the map. Make a new PR to update the map.

**Comment by Ghommie** [comment]
Oh right, the ancient goliath digs... it definitely should've a cooldown, and a limit to how much it can dig based on its surroundings (e.g. more than a couple sand objects in a 3x3 square).
